python/test/conftest.py

```diff
@@ -14,3 +16,27 @@ def test_environment():
     worksapce = os.environ.get("TIMEPLUS_WORKSPACE")
 
     return Environment().address(api_address).apikey(api_key).workspace(worksapce)
+
+
+@pytest.fixture
+def test_table(test_environment):
+    # Create a new stream
+    stream_name = "test_stream"
```

Comment:
to make a stream fixture, you can do

1. delete such stream, catch all exepctions, ignore error
2. create the stream

this will make sure a clean stream is created before reference